docs/server/source/production-deployment-template/architecture.rst

```diff
@@ -0,0 +1,69 @@
+Architecture of an IPDB Node
+============================
+
+An IPDB Production deployment is hosted on a Kubernetes cluster and includes:
+
+* NGINX, OpenResty, BigchainDB and MongoDB
+  `Kubernetes Services <https://kubernetes.io/docs/concepts/services-networking/service/>`_.
+* NGINX, OpenResty, BigchainDB, Monitoring Agent and Backup Agent
+  `Kubernetes Deployments <https://kubernetes.io/docs/concepts/workloads/controllers/deployment/>`_.
+* MongoDB `Kubernetes StatefulSet <https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/>`_.
+* External systems like `3scale <https://3scale.net>`_,
+  `MongoDB Cloud Manager <https://cloud.mongodb.com>`_ and the
+  `Azure Operations Management Suite
+  <https://docs.microsoft.com/en-us/azure/operations-management-suite/>`_.
+
+.. image:: ../_static/arch.svg
+
+
+We describe the role of each of these modules here.
```

Comment:
nit: Can we remove this line or rephrase? Or a subheading `Modules` or `Roles` that has sub-headings for each component.

Reply:
Removed.